src/views/Settings.vue

```diff
@@ -34,8 +34,8 @@
           <v-card-actions>
             <v-radio-group v-model="footnoteMethod">
               <v-radio label="නොපෙන්වන්න" value="hidden"></v-radio>
-              <v-radio label="click එබූ විට පෙන්වන්න" value="click"></v-radio>
-              <v-radio label="hover විට පෙන්වන්න" value="hover"></v-radio>
+              <v-radio label="ඔබන විට පෙන්වන්න" value="click"></v-radio>
+              <v-radio label="මතින් යනවිට පෙන්වන්න" value="hover"></v-radio>
```

Comment:
would be nice to include english (on click) (on hover) in brackets as well

Reply:
ඔබගේ යෝජනා යොදා ඇත.